src/components/LoginMenu.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+import FirebaseLogin from './FirebaseAuth';
+
+export default function ResponsiveDialog() {
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
```

Comment:
This can be shortened:
```suggestion
  const handleClickOpen = () => setOpen(true);
  const handleClose = () => setOpen(false);
```

Reply:
got it